testsuite/features/step_definitions/common_steps.rb

```diff
@@ -205,7 +205,9 @@
   output, _code = $server.run("python3 /tmp/#{file}")
   count_failures = output.to_s.scan(/false/).length
   output = output.join.to_s if output.respond_to?(:join)
-  raise "\nFound #{count_failures} failures in salt event log:\n#{output}\n" if count_failures.nonzero?
+  # Ignore the error if there is only the expected failure from min_salt_lock_packages.feature
+  ignore_error = (count_failures == 1) && output.include?('remove lock') && !$build_validation
```

Comment:
That kind of one-liners are not really readable.
No worries adding extra code lines, are free :D